src/Dumplie/Application/Metadata/MetadataAccessObject.php

```diff
@@ -28,35 +33,39 @@
     private $hydrator;
 
     /**
-     * @param Storage $storage
+     * @param Storage    $storage
+     * @param Schema     $schema
      * @param TypeSchema $typeSchema
-     * @param Hydrator $hydrator
+     * @param Hydrator   $hydrator
      */
-    public function __construct(Storage $storage, TypeSchema $typeSchema, Hydrator $hydrator)
+    public function __construct(Storage $storage, Schema $schema, TypeSchema $typeSchema, Hydrator $hydrator)
```

Comment:
it is but thats not a reason to put whole schema inside of it, name should  be enough